test/server/services/user.js

```diff
@@ -65,6 +68,66 @@ describe('User service tests', () => {
     });
   });
 
+  describe('Campaign User association', function() {
+    before((done) => {
+      this.userSaveParam = {
+        firstName: 'Jack',
+        lastName: 'J',
+        password: 'hatch1',
+        phoneNumber: '+144411144442',
+        email: 'Jack@gmail.com'
+      };
+
+      this.contactListParam = {
+        name: 'Millbrae Phone List 3'
+      };
+
+      this.campaignParams = {
+        name: 'testCampaign4',
+        title: 'Test4',
+        description: 'election',
+        status: 'draft',
+        script_id: 4,
+        contact_lists_id: 2
+      };
+
+      this.scriptParams = {
+        name: 'Script Name 4',
+        body: 'Script Body 4',
+        description: 'Script Description 4'
+      };
+
+      this.campaignUserParam = {
```

Comment:
I'd use the user and campaign ids you're adding as a part of the tests here rather than hard-coding in 3 - could get tricky if other users get added in or taken out of testing down the line.

Reply:
I'm going to have to store the ID somewhere as it comes back when I add the script.